data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaProducerProperties.java

```diff
@@ -53,8 +53,9 @@ public class KafkaProducerProperties {
     @JsonProperty("max_block")
     private Duration maxBlockMs = DEFAULT_MAX_BLOCK_MS;
 
+    // Initialize maxRequestSize to higher value to support larger messages
     @JsonProperty("max_request_size")
-    private int maxRequestSize = DEFAULT_MAX_REQUEST_SIZE;
+    private int maxRequestSize = 4*DEFAULT_MAX_REQUEST_SIZE;
```

Comment:
I think we should keep the `max_request_size` the same. There are a few reasons: 1) This change is going beyond Kafka recommendations (of 1MB). 2) This does change the existing behavior because it affects the topics on Kafka. 3) This may affect users without them clearly understanding the implications.